action.yaml

```diff
@@ -57,7 +57,11 @@ runs:
     - id: delay-checkout
       run: sleep ${{ inputs.delay }}
       shell: bash
-    - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+    # For performance, prefer the local GitHub mirror/repository when available
+    - name: "GitHub Checkout [continue-on-error enabled]"
```

Comment:
Adding in the `continue-on-error` isn't going to fix issues with brand new (empty) repo mirrors. The next step doesn't retry cloning if the repo cloned is non-existent or empty